### ISSUE TITLE
Unable to Compile on Windows 10 

### ISSUE THREAD

**Issue by KI5QMT** [SDRplay/RSPTCPServer#12]
I cloned the repo and ran a cmd in the directory.

Mkdir Build  Worked
Cd Build Worked
Cmake .. Failed with this in return:

C:\Users\lance\Downloads\RSPTCPServer\RSPTCPServer-0.1-beta\build>cmake ..
-- Building for: Visual Studio 17 2022
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.

-- Selecting Windows SDK version 10.0.22621.0 to target Windows 10.0.19045.
-- The C compiler identification is MSVC 19.38.33133.0
-- The CXX compiler identification is MSVC 19.38.33133.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.38.33130/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.38.33130/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE
64 bit compiler detected
-- libsdrplay not found.
CMake Error at CMakeLists.txt:11 (message):
  SDRPlay development files not found...

-- Configuring incomplete, errors occurred!

my SDR Play API is installed to C:\Program Files\SDRplay\API
I have installed CMake 
I installed Visual Studio 2019 and 2022.   

Any help is appreciated since the pre compiled binaries are no longer available from the sdrplay website.

**Comment by SDRplay** [comment]
Hi,That error message means that it cannot find the API - do you have it installed on your system?AndyOn 21 Dec 2023, at 14:41, KI5QMT ***@***.***> wrote:﻿
I cloned the repo and ran a cmd in the directory.
Mkdir Build  Worked
Cd Build Worked
Cmake .. Failed with this in return:
C:\Users\lance\Downloads\RSPTCPServer\RSPTCPServer-0.1-beta\build>cmake ..
-- Building for: Visual Studio 17 2022
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
Compatibility with CMake < 3.5 will be removed from a future version of
CMake.
Update the VERSION argument  value or use a ... suffix to tell
CMake that the project does not need compatibility with older versions.
-- Selecting Windows SDK version 10.0.22621.0 to target Windows 10.0.19045.
-- The C compiler identification is MSVC 19.38.33133.0
-- The CXX compiler identification is MSVC 19.38.33133.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.38.33130/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.38.33130/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE
64 bit compiler detected
-- libsdrplay not found.
CMake Error at CMakeLists.txt:11 (message):
SDRPlay development files not found...
-- Configuring incomplete, errors occurred!
my SDR Play API is installed to C:\Program Files\SDRplay\API
I have installed CMake
I installed Visual Studio 2019 and 2022.
Any help is appreciated since the pre compiled binaries are no longer available from the sdrplay website.

**Comment by KI5QMT** [comment]
yes I have installed sdrplay api version 3.07 on windows 10 it's directory is C:\programfiles\SDRPlay\API\

**Comment by KI5QMT** [comment]
does anyone have a already compiled exe

**Comment by KI5QMT** [comment]
yes I have installed sdrplay api version 3.07 on windows 10 it's directory is C:\Program Files\SDRplay\API
Every time i try to build using the sln file Visual Studio throws errors.
________________________________

**Comment by Sortam** [comment]
> does anyone have a already compiled exe

[KI5QMT](https://github.com/KI5QMT) Just in case you still need it - https://github.com/Sortam/RSPTCPServer/releases/tag/V.1.1.1
